vendor-extra/JatsContentBundle/tests/ViewConverter/ArticleTitleHeadingVisitorTest.php

```diff
@@ -131,6 +131,22 @@ public function it_sets_the_text_argument() : void
                             'context' => ['qux' => 'quux'],
                         ]
                     ),
+                    new View(
+                        null,
+                        [
+                            'node' => '/jats:article-title/jats:bold',
+                            'template' => null,
+                            'context' => ['qux' => 'quux'],
+                        ]
+                    ),
+                    new View(
+                        null,
+                        [
+                            'node' => '/jats:article-title/text()[3]',
```

Comment:
As I understand it, this will happen because there is whitespace that follows on from the `<jats:bold>` element.